### PR TITLE
Servlet sample

### PR DESCRIPTION
**Pull request by yasuflatland-lf**
Hi @gamerson , I just created a servlet sample. You'll see an error page when you access the servlet path, but I believe it's because any implementations have existed, am I correct? Please let me know if I need further work.

### CODE REVIEW

**Comment by lawrence-lee** [comment]
@yasuflatland-lf Are you able to investigate how to get the http servlet sample working? It would help us a lot.

**Comment by yasuflatland-lf** [comment]
@lawrence-lee , I've added README.markdown and described steps how it works. Please let me know if it's not clear enough.

File: gradle/extensions/http-servlet/README.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## Usage
+1. Set up Liferay Workspace.
+2. Go to ```${liferay_workspace_root}/modules``` and clone this repository
+3. Go to the root folder of this repository that you just cloned
+4. Start up your Liferay bundle
+5. Run ```gradle deploy``` and place the generated jar into ```${liferay_home}/deploy``` or if you have ```blade``` installed, run ```blade deploy```
+6. Sign in to Liferay as Administrator
+7. Navigate to Control Panel -> Configuration -> Server Administration -> Log Levels -> Add Category
+8. Configure ```com.liferay.blade.samples.httpservlet.BladeHttpServlet``` and the log level to ```INFO```
+9. Access to ```http://localhost:8080/o/blade/httpservlet``` You'll see error page (no actual implementations have existed yet) and info log on your console.
```
Comment (gamerson) [inline gradle/extensions/http-servlet/README.markdown]: @yasuflatland-lf  can you explain this error?  What does it mean that `no actual implementations have existed yet`?

Comment (yasuflatland-lf) [reply gradle/extensions/http-servlet/README.markdown]: @gamerson  You can actually access to the page, but it's plain Tomcat's page. What page is it suppose to be displayed? Maybe plain "It worked!" kind of page should be created and displayed?

Comment (lawrence-lee) [reply gradle/extensions/http-servlet/README.markdown]: @yasuflatland-lf I followed the instructions on the README, but on step 9, this is what i get in the console,
```18:07:56,848 WARN  [http-nio-8080-exec-8][code_jsp:172] {code="404", msg="ProxyServlet: /blade/httpservlet", uri=/o/blade/httpservlet}```

Then i get redirected to /web/guest/home

**Comment by yasuflatland-lf** [comment]
@lawrence-lee , @gamerson , Changed the code to generate texts.

File: gradle/extensions/http-servlet/src/main/java/com/liferay/blade/samples/httpservlet/BladeHttpServlet.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Liferay
+ */
+@Component(
+	immediate = true,
+	property = {
+		"dynamic.data.mapping.form.builder.servlet=true",
```
Comment (gamerson) [inline gradle/extensions/http-servlet/src/main/java/com/liferay/blade/samples/httpservlet/BladeHttpServlet.java]: Hey @yasuflatland-lf  what is this property for?

File: gradle/extensions/http-servlet/src/main/java/com/liferay/blade/samples/httpservlet/BladeHttpServlet.java [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+		"osgi.http.whiteboard.context.path=/",
+		"osgi.http.whiteboard.servlet.name=liferay.blade.samples.httpservlet.BladeHttpServlet",
```
Comment (gamerson) [inline gradle/extensions/http-servlet/src/main/java/com/liferay/blade/samples/httpservlet/BladeHttpServlet.java]: Does servlet name matter?

**Comment by yasuflatland-lf** [comment]
@gamerson @lawrence-lee  How about this?

**Comment by lawrence-lee** [comment]
Pushed to: https://github.com/gamerson/liferay-blade-samples/pull/407

**Pull request closed without merging**